include/zephyr/bluetooth/iso.h

```diff
@@ -967,6 +973,25 @@ struct bt_iso_unicast_tx_info {
 
 	/** The burst number */
 	uint8_t  bn;
+
+	/** The maximum SDU size in octets
+	 *
+	 * May be set to @ref bt_iso_unicast_tx_info.max_pdu for peripherals if unknown
+	 */
+	uint16_t max_sdu;
+
+	/** The SDU interval in microseconds
+	 *
+	 * May be set to  @ref BT_ISO_SDU_INTERVAL_UNKNOWN for if unknown.
+	 */
+	uint32_t sdu_interval;
+
+	/**
+	 * @brief Framing
+	 *
+	 * @ref BT_ISO_FRAMING_UNFRAMED for unframed and @ref BT_ISO_FRAMING_FRAMED for framed.
+	 */
+	uint8_t framing;
```

Comment:
Seems like framing is not set when V1 is raised. I'm wondering if we can leave that out from the public API. An application handles framed/unframed the same way. The only difference is that there is more freedom in terms of SDU interval at the cost of some increased transport latency

Reply:
Agreed. 

The central device will already know the framing anyways, and as you said, the peripheral can't or won't really treat it any different. I wonder why it was added to the event